Added new page explaining how to a question form on registration (#89)
front-commerce/developers.front-commerce.com: source/docs/magento2/add-question-on-registration.md

title: Add a question field when creating an account
---

> Here you will learn how to add a `Question` form that the user can fill when creating an account. You may want to check [this](/docs/magento2/add-new-attribute.html) page first, if you don't know how to add an attribute and create a Front-Commerce module.

Reviewer: ```suggestion
Here you will learn how to add a `Question` form that the user can fill when creating an account. You may want to check [how to add a new attribute docs](/docs/magento2/add-new-attribute.html) first, if you don't know how to add an attribute and create a Front-Commerce module.
```

Here is what the form initially looks like:

![Alt: screenshot of the register form with a `Question` field](./assets/without-question-screenshot.png)

(We will assume you already have done the [first](/docs/magento2/add-new-attribute.html) part.)

## Create a server module

> Create a server module, if you haven't already. In this example, ours we be called `acme`.

Reviewer: ```suggestion
Create a server module, if you haven't already. For the sake of this example, we will call it `acme`.
```

```bash
mkdir -p mymodule/server/modules/acme
```

> (We will assume you already have done the first part. Click [here](/docs/magento2/add-new-attribute.html) if you haven't already.)

Reviewer: ```suggestion
Follow the steps outlined in [add a module to Front Commerce](/docs/magento2/add-new-attribute.html#Add-the-module-to-Front-Commerce) docs
```
